### PR TITLE
Fixes #28072 - Expand the systemd unit

### PR DESCRIPTION
**Pull request by ekohl**
Note: this is untested.

### CODE REVIEW

**Comment by ekohl** [comment]
Since the original issue is already released I've created a new issue.

**Comment by adamruzicka** [comment]
Thanks @ekohl !

File: extra/cockpit/foreman-cockpit.service [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
+Description=Foreman Cockpit
+Documentation=https://theforeman.org
+After=network.target remote-fs.target nss-lookup.target
+
 [Service]
 Environment=XDG_CONFIG_DIRS=/etc/foreman/
 Environment=FOREMAN_COCKPIT_SETTINGS=/etc/foreman/cockpit/foreman-cockpit-session.yml
-ExecStart=/usr/libexec/cockpit-ws --no-tls --address 127.0.0.1 --port 9999
+Environment=ADDRESS=127.0.0.1
+Environment=PORT=9999
```
Comment (lzap) [inline extra/cockpit/foreman-cockpit.service]: This should have been FOREMAN_COCKPIT_ADDRESS instead, this is way too generic for a global environment space I think.

Comment (lzap) [reply extra/cockpit/foreman-cockpit.service]: Frankly we should probably move all variables from here into a separate file `/etc/foreman-cockpit.conf`.

Comment (lzap) [reply extra/cockpit/foreman-cockpit.service]: https://github.com/theforeman/foreman_remote_execution/pull/440

**Pull request merged**